bc/channel/models.py

```diff
@@ -67,13 +71,26 @@ class Channel(AbstractDateTimeModel):
     group = models.ForeignKey(
         "Group", related_name="channels", null=True, on_delete=models.SET_NULL
     )
+    access_token = models.TextField(
+        help_text="Access Tokens of the account that the bot is making the request on behalf of",
+        null=True,
+    )
+    access_token_secret = models.TextField(
+        help_text="Access Tokens Secret of the account that the bot is making the request on behalf of",
+        null=True,
```

Comment:
Should these be nullable instead of blankable?

Reply:
Thanks for pointing out this. It should be blankable. The Django docs advise not to use `null` on string-based fields.